libs/upstage/langchain_upstage/layout_analysis.py

```diff
@@ -112,8 +112,8 @@ def __init__(
                                      Defaults to None, in which case it will be
                                      fetched from the environment variable
                                      `UPSTAGE_API_KEY`.
-            use_ocr (bool, optional): Extract text from images in the document.
-                                      Defaults to False. (Use text info in PDF file)
+            use_ocr (bool, optional): Extract text from images in the document with OCR.
+                                      Defaults to None, Use default behavior of the API
```

Comment:
Please add the link of developer document of Upstage LA API Service with the more specific comment that you wrote in the readme file.

Reply:
I have applied the changes you suggested